docs/reference/esql/functions/replace.asciidoc

```diff
@@ -0,0 +1,16 @@
+[[esql-replace]]
+=== `REPLACE`
+This function substitutes the replacement string (3rd argument) for every occurrence of the regular expression (2nd argument) in the string (1st argument).
```

Comment:
Optional alternative: `The function substitutes in the string (1st argument) any match of the regular expression (2nd argument) with the replacement string (3rd argument).`
I'd find it easier to "read the function" as it flows.